### PR TITLE
Add defaultAzureCredential to eventHub blocks.

### PR DESCRIPTION
**Pull request by markosly**

### CODE REVIEW

File: gr-azure-software-radio/python/eventhub_sink.py [hunk 1]
```diff
@@ -112,15 +116,16 @@ def get_eventhub_producer_client(
         eventhub_producer_client = EventHubProducerClient.from_connection_string(
             connection_str, eventhub_name=eventhub_name)
 
-    elif authentication_method == "sas_token":
+    elif authentication_method == "sas":
```
Comment (jo-rivera) [inline gr-azure-software-radio/python/eventhub_sink.py:119]: thanks for fixing this bug

File: gr-azure-software-radio/python/integration_eventhub_sink.py [hunk 2]
```diff
@@ -96,13 +98,14 @@ def tearDown(self):
         self.tb = None
 
     def on_event(self, _partition_context, event):
-        msg = json.loads(list(event.body)[0])
-        print('Received the event: %s' % msg)
+        _ = json.loads(list(event.body)[0])
+        #print('Received the event: %s' % msg)
```
Comment (jo-rivera) [inline gr-azure-software-radio/python/integration_eventhub_sink.py:102]: should we remove this printout if is going to be commented out?

File: gr-azure-software-radio/python/integration_eventhub_sink.py [hunk 3]
```diff
@@ -96,13 +98,14 @@ def tearDown(self):
...
         self.num_rx_msgs += 1
         if self.num_rx_msgs == NUM_MSGS:
             self.eventhub_consumer.close()
 
     def test_round_trip_data_through_eventhub(self):
+        print("getting started")
```
Comment (jo-rivera) [inline gr-azure-software-radio/python/integration_eventhub_sink.py:108]: remove debug?

File: gr-azure-software-radio/python/integration_eventhub_source.py [hunk 4]
```diff
@@ -40,9 +43,8 @@ def tearDown(self):
         self.tb = None
         self.eventhub_producer.close()
 
-    def test_round_trip_data_through_eventhub(self):
+    def dont_test_round_trip_data_through_eventhub(self):
```
Comment (jo-rivera) [inline gr-azure-software-radio/python/integration_eventhub_source.py:46]: rename back to "test_*"

**Review by jo-rivera — CHANGES_REQUESTED** [review]

**Review by jo-rivera — APPROVED** [review]

File: gr-azure-software-radio/python/integration_eventhub_sink.py [hunk 5]
```diff
@@ -83,6 +83,9 @@ def setUp(self):
 
         self.eventhub_connection_string = os.getenv(
             'AZURE_EVENTHUB_CONNECTION_STRING')
+
+        self.eventhub_host_name = os.getenv(
+            'AZURE_EVENTHUB_HOST_NAME')
```
Comment (bquantump) [inline gr-azure-software-radio/python/integration_eventhub_sink.py:88]: Why'd this change?

File: gr-azure-software-radio/python/integration_eventhub_sink.py [hunk 6]
```diff
@@ -113,11 +115,59 @@ def test_round_trip_data_through_eventhub(self):
...
+    def test_round_trip_data_through_eventhub_default_creds(self):
+
+        creds = default_credentials.get_DefaultAzureCredential(enable_cli_credential=True,
+                                                               enable_environment=True,
+                                                               enable_managed_identity=True,
+                                                               enable_powershell=True,
+                                                               enable_visual_studio_code=True,
+                                                               enable_shared_token_cache=True,
+                                                               enable_interactive_browser=False)
+        print(dir(creds))
```
Comment (bquantump) [inline gr-azure-software-radio/python/integration_eventhub_sink.py:154]: remove

**Review by bquantump — APPROVED** [review]

**Pull request merged**